### PR TITLE
Korean Translation : web/css/_colon_user-valid/index.md

### PR DESCRIPTION
**Pull request by Kimhyuntae9665**
### Description

translate files/web/css/_colon_user-valid/index.md <== this one int korean 

### Motivation

because i translate this one , for korean and they gonna read this MDN Web Docs more easily

### CODE REVIEW

**Comment by Kimhyuntae9665** [comment]
Lint and review content files / lint-and-review-docs (pull_request_target)
I can not understand what is problem , Please Save me!!!!!!!!1

File: files/ko/web/css/_colon_user-valid/index.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+title: ":user-valid"
+slug: Web/CSS/:user-valid
+
```
[the remark this answers is not in the document]
Comment (Kimhyuntae9665) [reply files/ko/web/css/_colon_user-valid/index.md:4]: i gotta fix that one , and then 
Lint and review content files / lint-and-review-docs (pull_request_target) this error still working , how can i solve this one ?

**Comment by Kimhyuntae9665** [comment]
Please i can not understand what is problem of this one please save me 

Lint and review content files / lint-and-review-docs (pull_request_target)

**Comment by wisedog** [comment]
@Kimhyuntae9665  안녕하세요. 거의 린트에러 잡으셨긴 했는데 아래 2개만 더 수정하면 린트 통과할것 같습니다. 

1. 문서 마지막에 한 줄 더 추가하기
2. `en-US`를 `ko`로 수정하기

**Pull request closed without merging**